build/root/Makefile

```diff
@@ -309,6 +309,25 @@ clean:
 else
 clean: clean_meta
 	build/make-clean.sh
+	rm -rf Godeps/_workspace # Just until we are sure its gone
```

Comment:
This doesn't even even restore previous functionality. `/_tmp/` and `/_output/` need to be cleaned at a bare minimum.

Reply:
`/_output` is cleaned by the `kube::build::clean` call in `build/make-clean.sh`. You are correct about `/_tmp` however - thanks for point out :) If we decide to go forward with this, based on #52271, I'll update the diff.